rest/doc_api.go

```diff
@@ -605,9 +606,9 @@ func (h *handler) handleDeleteDoc() error {
 			return err
 		}
 	}
-	newRev, err := h.collection.DeleteDoc(h.ctx(), docid, revid)
+	newRev, cv, err := h.collection.DeleteDoc(h.ctx(), docid, revid)
 	if err == nil {
-		h.writeRawJSONStatus(http.StatusOK, []byte(`{"id":`+base.ConvertToJSONString(docid)+`,"ok":true,"rev":"`+newRev+`"}`))
+		h.writeRawJSONStatus(http.StatusOK, []byte(fmt.Sprintf(`{"id":`+base.ConvertToJSONString(docid)+`,"ok":true,"rev":"`+newRev+`", "current_version": {"source_id":"`+cv.SourceID+`", "version": %d}}`, cv.Value)))
```

Comment:
I didn't expect REST API changes to be part of the pull replication work - what's the reason this is required now?  (I don't think we've decided on things like the format of cv for REST API consumers)

Reply:
I also agree on this. I went ahead and did it to allow us to get the cv as well as the revid as this is needed for our blip tests to assert on both revid and cv when running the different subprotocol sub tests. See GetVersion that is called from WaitForVersion. 
An alternative is for me to remove these changes and then add a call to the _raw endpoint inside PutDoc which is then can be passed into DocVersionFromPutResponse to be unmarshaled?